news/_posts/core-weekly/2020-10-26-coreweekly-43-2020.md

```diff
@@ -0,0 +1,221 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 43 of 2020"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2020-10-26
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 19th to Sunday 25th of October 2020.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+### 1.7.7.0 Release Candidate 1 status
+
+The QA team is still running the test campaign on the second build for [**Release Candidate 1**](https://github.com/PrestaShop/PrestaShop/issues/18647).
+
+A major issue has been found in this build: [a change made from the Back-Office in an order that does not use the default currency results in the details of this order being displayed with the wrong currency symbol](https://github.com/PrestaShop/PrestaShop/issues/21566) [#21566](https://github.com/PrestaShop/PrestaShop/issues/21566).
+
+Since this issue is located into the View an Order Back-Office page, it has been decided that a third build, created after the bug has been fixed, will be necessary but this 3rd build test campaign will only target the Order Back-Office pages. The other areas of the software will not be tested again as they have already been scanned during the current test campaign.
```

Comment:
```suggestion
Since this issue is located into the "View an Order" Back-Office page, it has been decided that a third build, created after the bug has been fixed, will be necessary but this 3rd build test campaign will only target the "Order" Back-Office pages. The other areas of the software will not be tested again as they have already been scanned during the current test campaign.
```